packages/@okta/vuepress-site/docs/guides/sign-into-spa/require-authentication/blazorwebasm/reqautheverything.md

```diff
@@ -0,0 +1,12 @@
+To require authentication for the entire App add the following to `Imports.razor`
+
+```html
+@using Microsoft.AspNetCore.Authorization
+@attribute [Authorize]
+```
+
+Add this attribute to `Authentication.razor` component:
```

Comment:
This part is not clear, we have to tag with `Authorize` Imports.razor, but the Authentication page should remain anonymous? Can you refresh my mind why?

Reply:
@laura-rodriguez 
Authentication component starts actual auth process and should be accessible even if a user is not signed in.

Reply:
Right!